pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <groupId>com.exonum.binding</groupId>
   <artifactId>exonum-java-parent</artifactId>
-  <version>0.4-SNAPSHOT</version>
+  <version>0.1</version>
```

Comment:
Since it is a non-published pom (and mustn't be in sync with either ejb or lc), I think any will do.